ecms_base/themes/custom/ecms/ecms.theme

```diff
@@ -821,4 +853,18 @@ function ecms_preprocess_node__person(array &$variables) {
 
   // Comma separate all categories.
   $variables['categories'] = implode(', ', $variables['categories']);
+
+  // Build out additional fields array for twig.
+  if (!$node->field_person_additional_fields->isEmpty()) {
+    $paragraphs = $node->field_person_additional_fields->referencedEntities();
+    $variables['additional_fields'] = [];
+    foreach ($paragraphs as $paragraph) {
+      $field_label = $paragraph->get('field_person_field_label')->entity->getName();
+      $field_value = $paragraph->get('field_person_field_value')->getString();
```

Comment:
Were these tested for translated content? I think this needs 'getTranslation()' called somewhere